src/guide/render-function.md

```diff
@@ -136,9 +136,9 @@ The `h()` function is a utility to create VNodes. It could perhaps more accurate
 ```js
 // @returns {VNode}
 h(
-  // {String | Object | Function } tag
-  // An HTML tag name, a component or an async component.
-  // Using function returning null would render a comment.
```

Comment:
A function returning null is not really a special case, it's just a functional component. #1006 mentions that returning `null` creates a comment.